main_plot.py

```diff
@@ -56,7 +77,7 @@
 ]
 
 __plotters: dict[str, Plot[Data, Prediction, ConfidenceInterval, Title]] = {"heatmap": plot_results_in_heatmap_from_csv}
-
+__plotters_boxplot: dict[str, Plot[Data, Prediction, ConfidenceInterval, Title]] = {"boxplot": plot_results_in_boxplot_from_csv}
```

Comment:
Thinking about the intention behind the creation of `__plotters`. 
Would it be feasible to consider including the boxplot plotter within the same dictionary? 
This could potentially streamline the organization and simplify the implementation, for example: 
`__plotters: dict[str, Plot[Data, Prediction, ConfidenceInterval, Title]] = {"heatmap": plot_results_in_heatmap_from_csv, "boxplot": plot_results_in_boxplot_from_csv}`

Reply:
Done in the new PR